packages/x-components/.cypress-cucumber-preprocessorrc.json

```diff
@@ -1,4 +1,3 @@
 {
-  "nonGlobalStepDefinitions": true,
-  "stepDefinitions": "tests/e2e"
+  "stepDefinitions": "tests/e2e/**/*.ts"
```

Comment:
**nitpick**: increase `stepDefinitions` regex specificity (same as in the component `specPattern` config)

```suggestion
  "stepDefinitions": "tests/e2e/**/*.spec.ts"
```